RecoParticleFlow/PFSimProducer/plugins/PFSimParticleProducer.h

```diff
@@ -44,7 +50,7 @@ class PFSimParticleProducer : public edm::EDProducer {
   
   virtual void produce(edm::Event&, const edm::EventSetup&) override;
 
-  virtual void beginRun(const edm::Run& r, const edm::EventSetup & c) override;
+  virtual void beginRun(const edm::Run& r, const edm::EventSetup & c);
```

Comment:
`override` should not be deleted. If there is a reason to delete it, you need to add a comment explaining why this virtual function that overrides the original `beginRun` is not marked with `override`.